cli/src/tests/query_test.rs

```diff
@@ -1695,6 +1696,7 @@ fn test_query_matches_with_alternatives_and_too_many_permutations_to_track() {
         assert_eq!(cursor.did_exceed_match_limit(), true);
     });
 }
+*/
```

Comment:
This is only commented out temporarily; if we decide that we really want to go with this approach I'll clean up the test suite better.